### PR TITLE
Feature/why native app

### PR DESCRIPTION
**Pull request by abhijitkane**

### CODE REVIEW

File: 02_sending_and_viewing_responses/working_with_cookies.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Postman's [native apps](http://www.getpostman.com/apps) provide a Cookie Manager that lets you edit 
+cookies that are associated with each domain. To open the Cookie Manager, click the 'Manage Cookies' link under the 'Send' button.<br>
+
+[![](https://cloud.githubusercontent.com/assets/681190/16948881/e45d4816-4dd4-11e6-96e6-daadd28929a8.png)][1]
+
+This will open up the Cookie Manager window. The section on the left will show a list of domains which you have cookies defined. The section on the right will show you cookies associated with each domain.<br>
+
+[![](https://cloud.githubusercontent.com/assets/681190/16949440/74107796-4dd8-11e6-8b6e-4c7d52407fba.png)][2]
+
+### Create a cookie<br>
```
Comment (godfrzero) [inline 02_sending_and_viewing_responses/working_with_cookies.md]: `br`? Why not just use a blank line? 😕

File: 02_sending_and_viewing_responses/working_with_cookies.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+To add a new cookie, select the domain from the left pane, and enter the cookie string 
+(according to https://tools.ietf.org/html/rfc6265#section-4.1)
+in the input box in the bottom-right corner of the screen. <br>
+
+[![](https://cloud.githubusercontent.com/assets/681190/16949369/09af2460-4dd8-11e6-83db-1845afe54104.png)][3]
```
Comment (godfrzero) [inline 02_sending_and_viewing_responses/working_with_cookies.md]: This whole linking-to-image thing...it made sense when we displayed the thumbs and linked to the full-sized images. Not sure if it makes sense to link to the _same_ image.

File: 02_sending_and_viewing_responses/working_with_cookies.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Clicking the '+' button will add it to the app's cookie store 
+under the relevant domain.<br>
+
+[![](https://cloud.githubusercontent.com/assets/681190/16949373/0a169faa-4dd8-11e6-8c53-272ef7537733.png)][4]
+
+You can also add a cookie by setting a `Cookie` header, and sending a request to an appropriate URL in the main Postman window.
+
+
```
Comment (godfrzero) [inline 02_sending_and_viewing_responses/working_with_cookies.md]: Just a single empty line please 😄

File: 02_sending_and_viewing_responses/working_with_cookies.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+[![](https://cloud.githubusercontent.com/assets/681190/16949371/09ec4b4c-4dd8-11e6-8fc6-2dba4ad1ac07.png)][5]
+
+### Updating a cookie<br>
+To update an existing cookie, select the domain from the left pane, and the cookie you want to edit from the right pane. 
+You can edit any property except the cookie name, and hit '+' to update. Changing the name will add a new cookie.<br>
+
+[![](https://cloud.githubusercontent.com/assets/681190/16949370/09cb5e96-4dd8-11e6-9d28-ee21e501f875.png)][6]
+
+
```
Comment (godfrzero) [inline 02_sending_and_viewing_responses/working_with_cookies.md]: Just a single empty line here too 😮

**Comment by abhijitkane** [comment]
@godfrzero Removed <br> and links
it's still auto-linking, though

**Pull request merged**